src/main/java/mate/academy/dao/BookDaoImpl.java

```diff
@@ -0,0 +1,146 @@
+package mate.academy.dao;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import mate.academy.lib.Dao;
+import mate.academy.model.Book;
+import mate.academy.model.DataProcessingException;
+
+@Dao
+public class BookDaoImpl implements BookDao {
+    private static final int FIRST_QUESTION_MARK_INDEX = 1;
+    private static final int SECOND_QUESTION_MARK_INDEX = 2;
+    private static final String ID = "id";
+    private static final String TITLE = "title";
+    private static final String PRICE = "price";
+    private static final String ADD_LINE = " ";
+    private static final String INIT_DB_SQL = "src/main/resources/init_db.sql";
+    private static final String TAKE_BOOK_FROM_DB = "SELECT * FROM books WHERE id = ?";
+    private static final String DELETE_BOOK_FROM_DB = "DELETE FROM books WHERE id = ?";
+    private static final String TAKE_BOOKS_FROM_DB = "SELECT * FROM books";
+    private static final String SAVE_BOOK_IN_DB = "INSERT INTO books (title, price) VALUES (?, ?)";
+
+    public BookDaoImpl() {
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            String sqlCodeForTable = Files.readAllLines(Path.of(INIT_DB_SQL))
+                    .stream()
+                    .collect(Collectors.joining(ADD_LINE));
+
+            PreparedStatement statement = connection.prepareStatement(sqlCodeForTable);
+            statement.executeUpdate();
+        } catch (SQLException | IOException e) {
+            throw new DataProcessingException(e);
+        }
+    }
+
+    @Override
+    public Book create(Book book) {
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(SAVE_BOOK_IN_DB,
+                            Statement.RETURN_GENERATED_KEYS)) {
+
+            statement.setString(FIRST_QUESTION_MARK_INDEX,
+                    book.getTitle());
+            statement.setBigDecimal(SECOND_QUESTION_MARK_INDEX,
+                    book.getPrice());
+            int end = statement.executeUpdate();
```

Comment:
This name is not informative at all.
```suggestion
            int affectedRows = statement.executeUpdate();
```